### PR TITLE
#30 #31

### PR DESCRIPTION
**Pull request by YannisAm**
Made Restaurant and Menu classes and Added migrations

### CODE REVIEW

File: CRUDRecipeEF.BL.DL/Entities/Restaurant.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CRUDRecipeEF.BL.DL.Entities
+{
+    public class Restaurant
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
```
Comment (chrisK00) [inline CRUDRecipeEF.BL.DL/Entities/Restaurant.cs:12]: Required

**Review by chrisK00 — APPROVED** [review]

File: CRUDRecipeEF.BL.DL/Entities/Restaurant.cs (hunk 1 above)
Comment (YannisAm) [reply CRUDRecipeEF.BL.DL/Entities/Restaurant.cs:12]: I did it and when I added migration it poped this message "An operation was scaffolded that may result in the loss of data. Please review the migration for accuracy"
Do I continue?

Comment (chrisK00) [reply CRUDRecipeEF.BL.DL/Entities/Restaurant.cs:14]: Yeah continue

Comment (YannisAm) [reply CRUDRecipeEF.BL.DL/Entities/Restaurant.cs:14]: I added the required. I think its ready

File: CRUDRecipeEF.BL.DL/Entities/Menu.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CRUDRecipeEF.BL.DL.Entities
+{
+    public class Menu
+    {
+        public int Id { get; set; }
+        public List<Recipe> Recipes { get; set; } = new List<Recipe>();
```
Comment (chrisK00) [inline CRUDRecipeEF.BL.DL/Entities/Menu.cs:12]: We decided to add a name property right?

Comment (YannisAm) [reply CRUDRecipeEF.BL.DL/Entities/Menu.cs:12]: where do you want from me to add it?

Comment (chrisK00) [reply CRUDRecipeEF.BL.DL/Entities/Menu.cs:12]: Here in the menu since the restaurant has multiple menus we need to identify each menu somehow to the user

Comment (YannisAm) [reply CRUDRecipeEF.BL.DL/Entities/Menu.cs:12]: i added a name with required and pushed it

**Comment by chrisK00** [comment]
Looks good 👍

**Pull request merged**